popup/popup.js

```diff
@@ -576,10 +577,18 @@ Object.assign(handleEvent, {
             .then(() => msg.sendTab(tab.id, message));
         }
       })
-      .then(window.close);
+      .then(() => {
+        // Chromium needs help closing popup when opening new windows, not same window tabs
+        // FF needs help closing popup when opening same window tabs, not new windows
+        // edit URLs are ignored by pref - manager URls are passed to openURL() to check if tab exists
+        // which sends a message to close popup after determining it's not a different FF window
+        // because closing popup interferes with activating different windows in FF
+        if (this.dataset.href !== 'manage.html') window.close();
+      });
   },
 
   openManager(event) {
+    if (event.button === 2 && unsupportedURL) return;
```

Comment:
I think we can still open the manager without adding search params in this case.

Reply:
We can... but it doesn't work as advertised, so we should probably avoid that.

Reply:
Then I guess we should add an indicator that the right click doesn't work?

Reply:
Why? I removed the `Shift-click or right-click opens manager with styles applicable for current site` tooltip on unsupported URLs, because it's not true, and it doesn't do anything.